src/components/main.tsx

```diff
@@ -23,13 +17,9 @@ const Main = () => {
       <NavBar toggleMainWidth={toggleMainWidth} mini={minimizeSideBar} />
       <div className="content" id={mainId}>
         <Switch>
-          <Route exact path="/" component={Home} />
-          <Route path="/about" component={About} />
-          <Route path="/pybuild" component={PyBuild} />
-          <Route path="/cmd-word" component={CMDWord} />
-          <Route path="/memmap" component={MemMap} />
-          <Route path="/dte-file-creator" component={DTEFileCreator} />
-          <Route path="/miss-shob" component={MissShob} />
+          {routes.map((route) => (
```

Comment:
Careful! You're losing the `exact` prop of the home route

Reply:
delete/ resolve this if its done